webpack.config.js

```diff
@@ -1,8 +1,27 @@
+const DotEnvWebpack = require('dotenv-webpack');
 module.exports = (config, { isProd, isDev, isTest }) => {
-  /**
-   * Customize the webpack by modifying the config object.
-   * Consult https://webpack.js.org/configuration for more information
-   */
-
-  return config;
+    /**
+     * Customize the webpack by modifying the config object.
+     * Consult https://webpack.js.org/configuration for more information
+     */
+    // We dynamically change the path to the .env that contains the file corresponding to our profile
+    let envPath;
```

Comment:
This is now automatic - provided the name of the profile matches the envPath, i.e. `.env.${process.env.TWILIO_PROFILE}` would automatically be injected

Reply:
As this is already automatic, maybe we can just close this PR?